dapp/src/wagmiConfig.ts

```diff
@@ -21,21 +16,29 @@ const transports = chains.reduce(
   {},
 )
 
-const orangeKitUnisatConnector = getOrangeKitUnisatConnector(connectorConfig)
-const orangeKitOKXConnector = getOrangeKitOKXConnector(connectorConfig)
-const orangeKitXverseConnector = getOrangeKitXverseConnector(connectorConfig)
+async function getWagmiConfig() {
+  const {
+    getOrangeKitUnisatConnector,
+    getOrangeKitOKXConnector,
+    getOrangeKitXverseConnector,
+  } = await import("@orangekit/react")
```

Comment:
Wouldn't it work if we keep the import static as we used to? Maybe wrapping the loading with `getWagmiConfig` function is enough to solve the problem?